### ISSUE TITLE
[Visual Requirement – BotKit – Docs]: Luminosity Ratio is Less than 4.5:1 for the links present under “Building Blocks for Building Bots” header.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/botkit-website#2]
**User Experience:**
Low Vision users will get effected to navigate, if Luminosity Ratio is Less than 4.5:1 for the links present under “Building Blocks for Building Bots” header.

**Test Environment:**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://botkit.ai/
Browser: Edge
Tool Used: Colour Contrast Analyzer (CCA)

**Repro Steps:**
Step 1: Open the above URL in Edge browser.
Step 2: Open Colour Contrast Analyzer tool.
Step 3: Navigate to “Docs” link and select it.
Step 4: Navigate to the links present under “Building Blocks for Building Bots” header.
Step 5: Verify the Luminosity is greater than 4.5:1 or not.

**Actual Result:**
Luminosity Ratio is Less than 4.5:1 for the below links present under “Building Blocks for Building Bots” header.
V0.74
Up to date
MIT

**Expected Result:**
Luminosity Ratio should be Greater than or Equal to 4.5:1 for the below links present under “Building Blocks for Building Bots” header.
V0.74
Up to date
MIT

**MAS Reference Link**:
[](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}
![MAS1 4 3_BotKit_Docs](https://user-images.githubusercontent.com/48272005/54176128-8ef89000-44b3-11e9-8c2a-9083dafd7e28.png)
)

**Comment by benbrown** [comment]
The colors of these badges are defined by a third party service provider that manages these badges, and we cannot change them.

They are a very common element to include in this type of open source project.  Can this be given an exemption?

**Comment by awalia13** [comment]
yes if we can't edit these controls or these are external controls then we can take exception

**Comment by benbrown** [comment]
Do I need to formally request an exception on this?

**Comment by benbrown** [comment]
Request type: | Exception
-- | --
Leadership approval: | <List GPM or Dev Mgr. who has approved this request> NOTE: Attach approval communication to the request.  GPM or Dev Mgr should have conveyed in the communication that s/he understands the adverse impact to the experience.
Submitter (Accessibility Driver): | ​benbro@microsoft.com
UI / App name: | Botkit Website
User impact: | Some users with visual impairment may have a hard time reading the small color-coded badges that display the current version number and build status of the project. These are not critical values.
Repro steps: | See above
Provide work-around, if any: | No
Mitigating circumstances, if any: | n/a
Regression from past release: | no
Justification: | These badge images are created by third party providers. The colors are controlled by those third party providers and cannot be changed.
ETA to fix: | No ETA
